MoEPluginBase/MoEPluginUtils.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  MoEPluginUtils.swift
+//  MoEPluginBase
+//
+//  Created by Rakshitha on 23/06/22.
+//
+
+import Foundation
+import MoEngageInApps
+import MoEngageSDK
+
+public class MoEPluginUtils {
```

Comment:
@RakshithaAcharya Do you think this needs to be a singleton? Can we use protocol instead?

Reply:
@deepa-iOS-SDK  I have used protocol with static function.